catalog/app/components/JsonEditor/JsonEditor.js

```diff
@@ -24,23 +24,23 @@ const useStyles = M.makeStyles((t) => ({
   },
   inner: {
     display: 'flex',
-    maxHeight: t.spacing(42),
     overflow: 'auto',
   },
 }))
 
-function JsonEditor({
-  addRow,
-  changeValue,
-  className,
-  disabled,
-  columns,
-  jsonDict,
-  fieldPath,
-  makeAction,
-  onChange,
-  setFieldPath,
-}) {
+const JsonEditor = React.forwardRef((props, ref) => {
+  const {
+    addRow,
+    changeValue,
+    className,
+    disabled,
+    columns,
+    jsonDict,
+    fieldPath,
+    makeAction,
+    onChange,
+    setFieldPath,
+  } = props
```

Comment:
I started with `const Component = forwardRef({ ...props }, ref) => {}` and got additional indentation by prettier because arguments list (restructured props) is too long. So I decided to separate props destructuring.

Reply:
Adding named function removes this indentation problem